range/src/columns.rs

```diff
@@ -0,0 +1,8 @@
+#[derive(Default)]
+pub struct RangeCols<T> {
+    pub mult: T, // Multiplicity
```

Comment:
Seems unused, I take it the stark isn't quite finished?

Reply:
Yep, haven't written the constraints yet. It should be used there.